### ISSUE TITLE
`SA1402` Should not apply to file-scoped types

### ISSUE THREAD

**Issue by RenderMichael** [DotNetAnalyzers/StyleCopAnalyzers#3803]
The following code should not give a warning

```csharp
public sealed class C
{
}

file static class CHelpers
{
}
```

**Comment by bjornhellander** [comment]
Seems like a reasonable request to me, since this situation is why the file modifier was added

**Comment by 6bee** [comment]
Please merge PR #3879 to resolve this issue.

**Comment by 6bee** [comment]
There has been a ready-to-merge PR to resolve this issue, waiting for six month, that has not event been looked at. If maintenance for this project/repo has been abandoned, please mention it in the readme so people no longer need bother.
